site/app/controllers/forum/ForumController.php

```diff
@@ -1129,7 +1161,6 @@ public function getSplitPostInfo() {
         }
         return $this->core->getOutput()->renderJsonSuccess($result);
     }
-
```

Comment:
We generally put at least one newline between functions.  This whitespace change should be reverted.